penny_u_django/home/templates/home/index.html

```diff
@@ -3,10 +3,19 @@
 
 {% block content %}
 
-    <nav class="navbar navbar-light">
+    <nav class="navbar navbar-dark">
         <div class="m-auto d-flex justify-content-center">
             <span class="navbar-brand mb-0 h1">Penny University</span>
         </div>
+        <div class="float-right">
+            <a href="#" class="fa fa-facebook"></a>
+            <a href="#" class="fa fa-instagram"></a>
```

Comment:
Insta Link: https://www.instagram.com/officialpennyu/

Reply:
FYI if bootstrap doesn't have a way the easiest way to vertically align icons is to use line-height. Google "Vertical align icons line-height"